doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb

```diff
@@ -0,0 +1,1075 @@
+    "\n",
+    "After opening Bohrium Notebook, click the button `connect`, choose `bohrium-notebook` as image and `c4_m8_cpu` as computing resources. Wait a minute and you can get started.\n",
+    "\n",
+    "If you have any questions, please contact *[bohrium@dp.tech](mailto:bohrium@dp.tech)* for help. \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f4416190-479c-4b2f-9e08-33690bc4941d",
+   "metadata": {},
+   "source": [
+    "## Task\n",
+    "\n",
+    "> **Mastering the paradigm cycle of using DeePMD-kit to establish deep potential molecular dynamics models, and following a complete case to learn how to apply it to molecular dynamics tasks.**\n",
+    "\n",
+    "By the end of this tutorial, you will be able to:\n",
+    "\n",
+    "* Prepare the formataive dataset and running scripts for training with DeePMD-kit;\n",
+    "* Train, freeze, and test DeePMD-kit models;\n",
+    "* Use DeePMD-kit in Lammps for calculations;\n",
+    "\n",
+    "Work through this tutorial. It will take you 20 minutes, max!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e6587cb-ba6e-42ba-a139-0595fc7f79d7",
+   "metadata": {},
+   "source": [
+    "## Table of contents\n",
+    "\n",
+    "* Get tutorial data via git\n",
+    "* General Introduction\n",
+    "* Data preparation\n",
+    "* Prepare input script\n",
+    "* Train a model\n",
+    "* Freeze a model\n",
+    "* Test a model\n",
+    "* Run MD with LAMMPS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "adc6bbb1-a51b-421d-88f6-eb8eef79f432",
+   "metadata": {},
+   "source": [
+    "## Get tutorial data via git"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f493fdb7-7844-4152-8b8e-a4d9d1da58ff",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cloning into 'colombo-academy-tutorials'...\n",
+      "remote: Enumerating objects: 7164, done.\u001b[K\n",
+      "remote: Counting objects: 100% (174/174), done.\u001b[K\n",
+      "remote: Compressing objects: 100% (138/138), done.\u001b[K\n",
+      "remote: Total 7164 (delta 78), reused 71 (delta 32), pack-reused 6990\u001b[K\n",
+      "Receiving objects: 100% (7164/7164), 45.31 MiB | 3.75 MiB/s, done.\n",
+      "Resolving deltas: 100% (3378/3378), done.\n",
+      "Updating files: 100% (185/185), done.\n"
+     ]
+    }
+   ],
+   "source": [
+    "! if ! [ -e colombo-academy-tutorials ];then git clone https://gitee.com/deepmodeling/colombo-academy-tutorials.git;fi;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f98ce5f8-fb36-4e00-962f-028011801a53",
+   "metadata": {},
+   "source": [
+    "## General Introduction\n",
+    "This tutorial will introduce you to the basic usage of the DeePMD-kit, taking a gas phase methane molecule as an example. [DeePMD-kit's document](https://docs.deepmodeling.org/projects/deepmd/en/master/index.html \n",
```

Comment:
In this file, I prefer relative links to absolute links as people read this file inside the documentation.